### DISCUSSION TITLE
Possible to get green, not orange, when not actively charging(i.e. when my limit is reached) on the new MagSafe 3?

### DISCUSSION THREAD

**Discussion by Nimradd** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#464] [Q&A]
I'm on the new MacBook Pro with M1 Pro and I'm wondering if I can have the app override the status light on my MagSafe charger? Ideally for me I would like to have a green light when it's not actively charging, for example if it's reached my 80% limit. It doesn't seem to be a feature at the moment, but is this even possible in the future?
It's like the only feature not making this perfect for me, so I'm still in the boat whether or not to get the Pro version.

**Comment by MatthiasKerbl** [comment]
> I'm on the new MacBook Pro with M1 Pro and I'm wondering if I can have the app override the status light on my MagSafe charger? Ideally for me I would like to have a green light when it's not actively charging, for example if it's reached my 80% limit. It doesn't seem to be a feature at the moment, but is this even possible in the future? It's like the only feature not making this perfect for me, so I'm still in the boat whether or not to get the Pro version.

Hi @Nimradd,

It would definitely be a nice additional feature of AlDente. However, we don’t know if it is possible but we will look into it.

Cheers,
Matthias

**Comment by Fenderman2** [comment]
It would also great appreciate if this feature could be added.
Isn't the feasibility already shown here?
https://github.com/godly-devotion/charge-limiter

Thanks in advance, great app!

**Reply by MatthiasKerbl** [reply]
Hi @Nimradd,

This only works for Magsafe 2 MacBooks and not Magsafe 3.
But maybe it is possible.

Cheers,
Matthias

**Reply by OAG7** [reply]
I would like this too. MacBook Pro MAX. MagSafe 3.

**Comment by kentzh1** [comment]
Green indicates uncharged, orange indicates charging status, which will be more intuitive and easy to distinguish the current status. It would be great if it could be done.

**Comment by andrashatvani** [comment]
This is a feature worth paying for. Currently I don't use the MagSafe cable in the evenings as I find the amber light too aggressive. 
@MatthiasKerbl @davidwernhart
